File: task-03/SubTask2/file_io.c
#include <stdio.h>

int main() {
    FILE *infile, *outfile;
    char content[1000];

    infile = fopen("input.txt", "r");
    outfile = fopen("output.txt", "w");

    if (infile != NULL) 
    { 
        while (fgets(content, sizeof(content), infile) != NULL) 
        {
            fputs(content, outfile);
        }
        fclose(infile);
        fclose(outfile);
    }
    return 0;
}

File: task-03/SubTask2/file_io.java
import java.nio.file.*;

public class FileIO 
{
    public static void main(String[] args) 
    {
        try 
        {
            String content = Files.readString(Path.of("input.txt"));
            Files.writeString(Path.of("output.txt"), content);
        } 
        catch (Exception e) 
        {
            e.printStackTrace();
        }
    }
}
